public/firebase-messaging-sw.js

```
@@ -0,0 +1,72 @@
+    self.registration.showNotification(notificationTitle, notificationOptions);
+  }
+});
+
+
+// 알림 클릭 이벤트 처리
+self.addEventListener('notificationclick', function(event) {
+  console.log('Notification clicked:', event);
+
+  event.notification.close();
+
+  const urlToOpen = event.notification.data?.url || 'https://mungtivity.vercel.app';
```

ESLint

[error] Insert `⏎···`

(prettier/prettier)